src/OrchardCore/OrchardCore.DisplayManagement/Html/CssOrchardHelper.cs

```diff
@@ -86,4 +87,38 @@ private static IEnumerable<string> Combine(string optionClasses, string[] additi
 
         return additionalClasses.Concat(new[] { optionClasses });
     }
+
+    private static IEnumerable<string> GetPartWrapperCssClasses(ContentTypePartDefinition partDefinition)
+    {
+        yield return "content-part-wrapper";
+        yield return $"content-part-wrapper-{partDefinition.PartDefinition.Name.HtmlClassify()}";
+
+        if (partDefinition.IsNamedPart())
+        {
+            yield return $"content-part-wrapper-{partDefinition.Name.HtmlClassify()}";
+        }
+    }
+
+    private static IEnumerable<string> GetFieldWrapperCssClasses(ContentPartFieldDefinition fieldDefinition)
```

Comment:
This method should `return GetWrapperCssClasses("the", "field", "wrapper", "class", "names")`

Reply:
same as previous comment